LICENSE.md

```diff
@@ -0,0 +1,11 @@
+# License for the specifications
+
+The ABI document sources themselves are not dependent on any assets outside of their own directory and all have their own license file included in the directory. The ABI documents are currently licensed under the Creative Commons Attribution-ShareAlike 4.0 International License + grant of Patent License. Contributions to these files are accepted under the same license.
+
+# License for the legacy releases
+
+All the ABI documents in the `legacy-specifications' folder are licensed under an Arm proprietary license. These predate the new license of the current versions of the documents. The license is contained within the documents themselves, so for more information, consult the individual documents.
```

Comment:
Typo: right after legacy-specifications, there's a single quote where it should have another backtick.